client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```
@@ -74,6 +74,10 @@
 			row-gap: 48px;
 			margin: 0 0 30px;
 
+			@include break-medium {
+				grid-template-columns: 1fr 1fr;
```

I think the 64px gap is too big for mobiles. 

```suggestion
				grid-template-columns: 1fr 1fr;
				column-gap: 32px;
```

---

@dubielzyk you can maybe comment to that

---

Indeed too big. We should drop it down to 32px fairly quickly (tablet) and then 20px on mobile.

---

@yansern Let's have the column gap go:

- 20px
- 32px tablet
- 64px desktop